src/presenter/board-presenter.js

```diff
@@ -0,0 +1,40 @@
+import FilterView from '../view/filter-view.js';
+import EventListView from '../view/event-list-view.js';
+import SortView from '../view/sort-view.js';
+import EventPointView from '../view/event-point-view.js';
+import EventEditView from '../view/form-edit-view.js';
+import EventsBoardView from '../view/events-board-view.js';
+import TripInfoView from '../view/trip-info-view.js';
+import {render} from '../render.js';
+import {RenderPosition} from '../render.js';
+
+const siteHeaderElement = document.querySelector('.page-header');
+const tripMainElement = siteHeaderElement.querySelector('.trip-main');
+const siteMainElement = document.querySelector('.page-body__page-main');
+
+export default class BoardPresenter {
+  boardComponent = new EventsBoardView();
+  sortComponent = new SortView();
+  eventListComponent = new EventListView();
+  filterComponent = new FilterView();
+  tripInfoComponent = new TripInfoView();
+  formEditComponent = new EventEditView();
+  eventPointComponent = new EventPointView();
+
+  constructor({container}) {
+    this.container = container;
+  }
+
+  init() {
+    render(this.filterComponent, tripMainElement);
+    render(this.tripInfoComponent, tripMainElement, RenderPosition.AFTERBEGIN);
+    render(this.sortComponent, this.container);
+    render(this.boardComponent, this.container);
+    render(this.eventListComponent, siteMainElement);
+    render(this.formEditComponent,this.boardComponent.getElement());
+
+    for (let i = 0; i < 3; i++) {
+      render(this.eventPointComponent, this.boardComponent.getElement());
+    }
```

Comment:
сейчас проблема в том что ты отрисовываешь один и тот же элемент, внутри render вызывается insertAdjacentElement, а он проверку делает, если этот элент уже в дом дереве, повторно его не вставляет, тебе нужно именно три оттдельных компонента, пока хватит сделать что-то типа 
```suggestion
    for (let i = 0; i < 3; i++) {
      render(new EventPointView(), this.boardComponent.getElement());
    }
```

Reply:
но сейчас есть проблемка с разметкой, сравни внимательно с макетами, потому что у тебя сейчас li внутри дива лежит, где то ul потерялся )
<img width="1096" alt="Снимок экрана 2024-06-13 в 21 05 53" src="https://github.com/htmlacademy-ecmascript/1044677-big-trip-2/assets/52581179/e97d0311-7635-4694-be82-658a4a4db67d">